app/Http/Controllers/CampaignController.php

```diff
@@ -5,20 +5,42 @@
 use App\Http\Requests\CampaignRequest;
 use App\Models\Campaign;
 use App\Models\SocialPlatform;
+use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Support\Facades\Auth;
-use Illuminate\Support\Facades\Redirect;
-use PhpParser\Node\Expr\NullsafeMethodCall;
+use Illuminate\Support\Facades\DB;
 
 class CampaignController extends Controller
 {
+    /**
+     * Campaign paginate
+     *
+     * @return \App\Models\Campaign
+     */
+    public function campaignPaginate()
+    {
+        $noItems = 2;
```

Comment:
Please make an enum for $noItems = 2;
ex:
config/const.php 
return [
'campaign_item' => 2
];
or

Campaign Model
const ITEMS = 2;

Reply:
Noted, sir.